assets.src/src/styles/time_sessions_table.scss

```diff
@@ -6,4 +6,8 @@
   .timer-session-table-actions {
     text-align: right !important;
   }
+
+  .gap-marker {
+    border-top: 1px solid red;
+  }
```

Comment:
Red is the color for errors. Gaps are not errors. Gaps are normal breaks. Please use warn or info colors.